apis/r/R/SOMADataFrame.R

```diff
@@ -42,6 +43,16 @@ SOMADataFrame <- R6::R6Class(
       # Parse the tiledb/create/ subkeys of the platform_config into a handy,
       # typed, queryable data structure.
       tiledb_create_options <- TileDBCreateOptions$new(platform_config)
+      #print(str(tiledb_create_options$to_list()))
```

Comment:
remove please

Reply:
Will do in a few hours once I synced with @mojaveazure.  The comments provide context.